network/gossip/libp2p/test/testUtil.go

```diff
@@ -0,0 +1,191 @@
+package test
```

Comment:
the new and improved `helper` to setup the network for network related unit tests